docs/inference.md

```diff
@@ -79,6 +79,14 @@ To run the code infilling example:
 python examples/code_llama/code_infilling_example.py --model_name MODEL_NAME --prompt_file examples/code_llama/code_infilling_prompt.txt --temperature 0.2 --top_p 0.9
 
 ```
+To run the 70B Instruct model example run the following, it asks for system and user prompt to instruct the model:
+
+```bash
+
+python code_instruct_example.py --model_name codellama/CodeLlama-70b-Instruct-hf
```

Comment:
add `examples/code_llama/` before code_instruct_example.py ?

also , should we mention that it'd take about 10 minutes and 130GB free space to download all the shard safetensors model files?

Reply:
I got the following error when running the command above (as well as `python examples/code_llama/code_completion_example.py --model_name codellama/CodeLlama-7b-Instruct-hf --prompt_file examples/code_llama/code_completion_prompt.txt --temperature 0.2 --top_p 0.9`) after all the model files were downloaded (notice my transformers and torch versions are shown to be ok and don't need to be upgraded):
<img width="1611" alt="image" src="https://github.com/facebookresearch/llama-recipes/assets/535090/384db282-d14e-47c5-993b-1931095cdf97">

This is how I installed the packages inside a new conda environment with python 3.11.7:
```
pip install git+https://github.com/huggingface/transformers
pip install --extra-index-url https://download.pytorch.org/whl/test/cu118 llama-recipes
```

Reply:
sounds like an [issue](https://github.com/huggingface/transformers/issues/28877) on HF side. we can disable the` --fast_kernel False` for now until the fix come along.

Reply:
This has been resolved with `attn_implementation="eager`, all references to better_transformer is now updated.